docs/securing-control-plane.md

```diff
@@ -0,0 +1,154 @@
+# Securing Control Plane
+
+All API communication between Antrea control plane components is encrypted with
+TLS. The TLS certificates that Antrea requires can be automatically generated.
+You can also provide your own certificates. This page explains the certificates
+that Antrea requires and how to configure and rotate them for Antrea.
+
+## Table of Contents
+
+- [What certificates are required by Antrea](#what-certificates-are-required-by-antrea)
+- [How certificates are used by Antrea](#how-certificates-are-used-by-antrea)
+- [Providing your own certificates](#providing-your-own-certificates)
+  - [Using kubectl](#using-kubectl)
+  - [Using cert-manager](#using-cert-manager)
+- [Certificate rotation](#certificate-rotation)
+
+## What certificates are required by Antrea
+
+Currently Antrea only requires a single server certificate for the
+antrea-controller API server endpoint, which is for the following communication:
+- The antrea-agents talks to the antrea-controller for fetching the computed
+ NetworkPolicies
+- The kube-aggregator (i.e. kube-apiserver) talks to the antrea-controller for
+ proxying the antctl's troubleshooting requests
+
+Antrea doesn't require client certificates for its own components as it
+delegates authentication and authorization to the Kubernetes API, using 
+Kubernetes [ServiceAccount token](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens)
+for client authentication.
+
+## How certificates are used by Antrea
+
+By default, antrea-controller generates self signed
+certificate. You can override the behavior by [providing your own certificate](#providing-your-own-certificates).
+In either way, the antrea-controller will distribute the CA certificate as a
+ConfigMap named `antrea-ca` in the `kube-system` Namespace and inject it into
+the APIServices resources created by Antrea in order to allow its clients (i.e.
+antrea-agent, kube-apiserver) to trust itself.
+
+Typically, clients that wish to access the antrea-controller API can
+authenticate server by validating against the CA certificate published in the
+`kube-system/antrea-ca` ConfigMap.
+
+## Providing your own certificates
+
+Since Antrea v0.7.0, you can provide your own certificates to Antrea. To do so,
+you must set the `selfSignedCert` field of `antrea-controller.conf` to `false`,
+so that the antrea-controller will read the certificate key pair from the
+`kube-system/antrea-controller-tls` Secret.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: antrea
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-controller.conf: |
+    selfSignedCert: false
+```
+
+You can generate the required certificates manually, or through
+[cert-manager](https://cert-manager.io/docs/). In either way, the certificate
```

Comment:
You switch from "certificates" to "certificate" which I find a bit confusing.
Maybe use "certificate" consistently (like you did before this section), which implies the "TLS certificate"

Reply:
I changed some "certificates" to "certificate" but kept some when making a general reference like "What certificates are required by Antrea", "Providing your own certificates". Please check if it's proper now.